util/test/perf/perfgraph.js

```diff
@@ -379,19 +382,25 @@ function setupScreenshotToggle(g, graphInfo, screenshotToggle) {
   }
 }
 
-
 // Setup the close graph button
 function setupCloseGraphToggle(g, graphInfo, closeGraphToggle) {
   closeGraphToggle.style.visibility = 'visible';
 
   closeGraphToggle.onclick = function() {
     var checkBox = getCheckboxForGraph(g);
     checkBox.checked = false;
-
-    // TODO instead of completely redrawing with displaySelectedGraphs() we
-    // could just remove the divs associated with the graph being removed to
-    // speed up this operation and make it less "jittery"
-    displaySelectedGraphs();
+    $(g.divs.div).hide();
```

Comment:
I'm ok with this, but it might be a little cleaner to do 

```js
for (var div in g.divs) {
  if (g.divs.hasOwnProperty(div)) {
    div.hide()
  }
}
```